devon4j-app/index.asciidoc

```diff
@@ -10,33 +10,91 @@ After completing this scenario, you will have learned how to generate Java appli
 
 More information about devon4j on https://devonfw.com/website/pages/docs/devon4j.asciidoc.html
 
-== Project structure
-The structure of a devon4j application is divided into the following modules:
+====
+====
+### Install devonfw-ide
+To create devon4j application it is necessary to have devonfw-ide installed on your machine.For this tutorial purpose we will have it installed here.
```

Comment:
That's actually not true. You can create a devon4j application without devonfw ide. That's important to be clear here as people think already that they have to take everything when using devonfw.
Please see also there is a missing whitespace after the dot.